app/controllers/state_file/questions/nj_college_dependents_exemption_controller.rb

```diff
@@ -0,0 +1,19 @@
+module StateFile
+  module Questions
+    class NjCollegeDependentsExemptionController < QuestionsController
+      include ReturnToReviewConcern
+
+      def self.show?(intake)
+        intake.dependents.count(&:under_22?).positive?
+      end
+
+      def form_params
+        params
+          .require(:state_file_nj_college_dependents_exemption_form)
+          .permit(
+            form_class.attribute_names +
+              [{ dependents_attributes: [:id, :nj_dependent_attends_accredited_program, :nj_dependent_enrolled_full_time, :nj_dependent_five_months_in_college, :nj_filer_pays_tuition_for_dependent] }])
```

Comment:
[dust] P long line. Can we break this up a little to make it easier to scan? Not a major thing though.